.github/workflows/build-publish-pr.yml

```diff
@@ -53,6 +54,50 @@ jobs:
           dockerhub_username: ${{ secrets.DOCKERHUB_READONLY_USERNAME }}
           dockerhub_password: ${{ secrets.DOCKERHUB_READONLY_PASSWORD }}
 
+      - name: Get PR labels
+        id: pr-labels
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          RESPONSE=$(gh pr view ${PR_NUMBER} --json labels)
+          # Check if the labels command was successful
+          if [[ $? -ne 0 ]]; then
+            echo "Error fetching labels"
+            exit 1
+          fi
+          echo "RESPONSE=${RESPONSE}"
+          LABELS=$(echo "$RESPONSE" | jq -r '.[].name')
```

Comment:
the output is something like this I think:

```
{
  "labels": []
}
```

Reply:
nvm - all good here :)

Reply:
so like this?
```
jq -r '.labels.[].name'
```
would the output be stored in `LABELS` as a list?

Reply:
Ah that works, fixed in b800601a52253c392d9e636160d66843f14354b5 -- good catch, thanks!

Reply:
I just tested it with multiple labels and they are all on one string and space separated which should work with:

```
        if: contains(steps.pr-labels.outputs.labels, 'crib')
```